indico/web/client/js/react/components/DateRangePicker.jsx

```diff
@@ -0,0 +1,69 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import 'react-dates/initialize';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {DateRangePicker as DRP} from 'react-dates';
```

Comment:
![](https://i.kym-cdn.com/entries/icons/mobile/000/001/313/DERP.jpg)

what about `ReactDateRangePicker` instead? then you also don't need to silence eslint below